tests/phpunit/tests/block-supports/duotone.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/**
+ * Test the block WP_Duotone class.
+ *
+ * @package WordPress
+ */
+
+class WP_Duotone_Test extends WP_UnitTestCase {
+	/**
+	 * Cleans up CSS added to block-supports from duotone styles. We neeed to do this
+	 * in order to avoid impacting other tests.
+	 */
+	public static function wpTearDownAfterClass() {
+		WP_Style_Engine_CSS_Rules_Store::remove_all_stores();
+	}
+
+	/**
+	 * Tests whether the duotone preset class is added to the block.
+	 *
```

Comment:
```suggestion
	 * @ticket 58555
	 *
```

I think we could also add `@ticket 58555`, referencing the [ticket](https://core.trac.wordpress.org/ticket/58555) of this PR